src/Domains/Souk/Services/B2BConfigurationService.php

```diff
@@ -42,4 +49,27 @@ public static function hasGlobalCompany(
 
         return false;
     }
+
+    public static function sendNotificationToUsers(AppInterface $app, Companies $company, string $templateName, Users $user, array $data = [])
+    {
+        if ($b2bCompany = B2BConfigurationService::getConfiguredB2BCompany($app, $company)) {
+            $users = $b2bCompany->users;
```

Comment:
@jesusantguerrero use UserRoleNotificationService

Reply:
I'll have to check, because sometimes that app, will be b2c but I need the users of b2b